package.json

```diff
@@ -66,7 +66,8 @@
     "@types/dom-mediacapture-record": "^1.0.4",
     "detect-browser": "^4.7.0",
     "protobufjs": "~6.8.8",
-    "resize-observer": "^1.0.0"
+    "resize-observer": "^1.0.0",
+    "uuid": "^8.2.0"
```

Comment:
Chime SDK doesn't need to depend on the `uuid` library. Update only in the integration and demo's package.json.

Reply:
Fixed.